src/app/shell/personal-cabinet/personal-cabinet.component.html

```diff
@@ -1,16 +1,52 @@
 <div class="container">
-  <h1 class="title">{{userRole === 'provider' ? 'КАБІНЕТ НАДАВАЧА' : userRole === 'admin' ? 'КАБІНЕТ ТЕХ. АДМІНА' : 'КАБІНЕТ КОРИСТУВАЧА'}}</h1>
+  <h1 class="title">
+    {{
+      userRole !== 'provider'
+      ? providerUserRole[userRole] 
+      : providerRole[(subrole$ | async)]
+    }}
+  </h1>
+
   <nav class="nav" mat-tab-nav-bar>
-    <a mat-tab-link [routerLinkActive]="['active']" [routerLink]="'./config'">ОСОБИСТА
-      ІНФОРМАЦІЯ</a>
-    <a *ngIf="userRole !== Role.techAdmin" mat-tab-link [routerLinkActive]="['active']" [routerLink]="'./' + userRole +'/' + 'info'">ІНФОРМАЦІЯ ПРО
-      {{ roles[userRole] | uppercase }}</a>
-    <a *ngIf="userRole !== Role.techAdmin" mat-tab-link [routerLinkActive]="['active']" [routerLink]="'./workshops'">{{ userRole === Role.parent ? 'МОЇ ГУРТКИ' : 'ГУРТКИ' }}</a>
-    <a *ngIf="userRole !== Role.techAdmin" mat-tab-link [routerLinkActive]="['active']" [routerLink]="'./applications'">ЗАЯВИ</a>
-    <a *ngIf="userRole == Role.provider" mat-tab-link [routerLinkActive]="['active']" [routerLink]="'./administration/all'">АДМІНІСТРУВАННЯ</a>
-    <a *ngIf="userRole === Role.parent" mat-tab-link [routerLinkActive]="['active']"
-      [routerLink]="'./parent/favorite'">УЛЮБЛЕНЕ</a>
+    <a mat-tab-link [routerLinkActive]="['active']" [routerLink]="'./config'"
+      >ОСОБИСТА ІНФОРМАЦІЯ</a
+    >
+    <a
+      *ngIf="userRole !== Role.techAdmin"
```

Comment:
use ng-container directive to merge conditions:
`<ng-container *ngIf="userRole !== Role.techAdmin"> A... B... C...</ng-container>
<ng-container *ngIf="userRole === Role.techAdmin"> D... E... F...</ng-container>`